packages/playground/src/components/smtp_server.vue

```diff
@@ -20,17 +20,23 @@
       <input-validator
         :value="$props.modelValue.username"
         :rules="[
-          validators.required('Email is required.'),
-          validators.isEmail('Please provide a valid email address.'),
-          v => {
-            return isDiscourse ? undefined : validators.isEmail('Please provide a valid email address.')(v);
+          validators.required('Email or Username is required.'),
+          validators.minLength('Username must be at least 2 characters.', 2),
+          validators.maxLength('Username must be at least 50 characters.', 50),
```

Comment:
we shouldn't use the `at least` on max lenghth

Reply:
also is this validation on user name only? what if the email exceeds the 50 char ? the length validation message will appear

Reply:
done